statsmodels/tsa/tests/test_holtwinters.py

```diff
@@ -341,6 +341,7 @@ def test_debiased():
     assert np.any(res.fittedvalues != res2.fittedvalues)
 
 
+@pytest.mark.smoke
 @pytest.mark.parametrize('trend', TRENDS)
 @pytest.mark.parametrize('seasonal', SEASONALS)
 def test_float_boxcox_smoke(trend, seasonal):
```

Comment:
I don't know what this test does, but it doesn't look like a smoke test

Reply:
@bashtage this looks like yours.  Is this mark appropriate here?  If not, can the "smoke" be taken out of the name?

Reply:
This is effectively a smoke test.  It only checks that the set value is actually set, nothing about results.